### DISCUSSION TITLE
Update…

### DISCUSSION THREAD

**Discussion by DariBer** [daglem/reDIP-SID#11] [Q&A] [answered]
I haven't seen any updates for a long time now, so just wonder is this project still in development or is abandoned?

**Comment by daglem** [comment]
I'll keep working on this whenever I find the time. Why do you ask?

**Reply by DariBer** [reply]
I'm interested in the project and would like to buy one to test and if they are any good I would start buy them when I repair a C64. 

Of course the price is a factor if I will buy them or not.

**Reply by daglem** [reply, marked as answer]
I'm quite happy with how it has turned out so far! Regarding emulation quality, it's comparable to reSID 0.16, except that emulation of the digital logic is, as far as I know, 100% correct. It still leaves something to be desired in the MOS6581 filter department (the MOS8580 filter is fine), and also lacks real-time configurability. I'm planning to address these areas in my own time, aiming for a "perfect" SID replacement.

Unfortuately the price for the FPGA has risen dramatically during the last few years, and the audio codec doesn't come for free either. BOM price per board for a batch of 100 is currently well north of $20, and that's not including the cost of PCBs and assembly. It is quite possible to order boards from a PBCA manufacturer (which is what I did for prototypes - it'll cost you), and if you have the skills, equipment, and patience, it is also possible to assemble boards manually (this has been done by others). I'm not sure I'll ever attempt to manufacture boards for sale, since I suspect it won't be worth my while. As it stands, the selling price would probably end up in the same ballpark as the FPGASID ($80), and I believe that's a quite limited market :sweat_smile:

If you want something less expensive, you'll have to go for an MCU based solution. Quite capable MCUs can go for 1/10 of the price for the FPGA in the reDIP SID, and so the final product will consequently be much cheaper. I don't have any experience with such SID replacements, but perhaps the open source [SIDKick pico](https://github.com/frntc/SIDKick-pico) can fit the bill?
